core/Console.php

```diff
@@ -34,11 +34,23 @@ public function __construct()
 
         $this->getDefinition()->addOption($option);
 
+        $option = new InputOption('xhprof',
+            null,
+            InputOption::VALUE_NONE,
+            'Enable profiling with XHProf'
+        );
+
+        $this->getDefinition()->addOption($option);
```

Comment:
In that case, can we echo a warning or throw an exception when trying to profile w/ development mode is enabled (would do this when calling setupProfilerXhprof for the main run)?